source/core/src/main/com/csse3200/game/inventory/items/potions/TimedUseItem.java

```diff
@@ -0,0 +1,96 @@
+package com.csse3200.game.inventory.items.potions;
```

Comment:
Should refactor this to only create it's own game time instance (shouldn't be part of subclass constructors). Then just create a getGameTime function for testing purposes.

Reply:
And duration should be specific to each item, not something for all timed use items